src/persistence/historykeeper.cpp

```diff
@@ -209,9 +211,13 @@ QString HistoryKeeper::getHistoryPath(QString currentProfile, int encrypted)
 
     const Profile* profile = Nexus::getInstance().getProfile();
     if (encrypted == 1 || (encrypted == -1 && profile->isEncrypted()))
+    {
         return baseDir.filePath(currentProfile + ".qtox_history.encrypted");
+    }
     else
```

Comment:
What the reason of `else` if we use `return` in both cases?
Also, it can be simplified:
```
QString path = currentProfile + QStringLiteral(".qtox_history")
if (encrypted == 1 || (encrypted == -1 && profile->isEncrypted())) 
{
    path += QStringLiteral(".encrypted");
}

return baseDir.filePath( + ext);
```